apis/apiextensions/v1alpha1/composition_types.go

```diff
@@ -313,7 +313,7 @@ type CompositionStatus struct {
 // type is created with reference to the composition.
 // +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:scope=Cluster,categories=crossplane
+// +kubebuilder:resource:scope=Cluster,categories=crossplane,shortName=xmp
```

Comment:
What does XMP stand for here?

Reply:
"Xompisition". We replace the C in "Composite Resource" and "Composite Resource Definition" with an X when we abbreviate them - i.e. XR, XRD - to distinguish them from upstream's CR and CRD. I originally went with "cmp" as the short name here, but it felt strange to have "xr, xrd, and cmp". That said, "xmp" feels strange too.

Reply:
I don't mind it, but I do think we should use it in the same places we use XR and XRD if we are going to use it at all :+1:

Reply:
I didn't intend "XMP" to be an official abbreviation for composition, but rather an informal a shorthand for `kubectl get composition`.  Similar to how `kubectl get rs` exists for replicasets, but I don't think they're formally abbreviated to "RS" in the documentation. I'll just remove the short name for now.